### PR TITLE
Latex mig

### PR DESCRIPTION
**Pull request by andrewkern**
moving the manuscript over to latex format here. this PR includes changes to the Makefile, the references, and adds a `.tex` file which should is _now_ the doc to add. I didn't take the liberty of deleting the now deprecated `.md` file, but we could do that as well if people like

### CODE REVIEW

File: adding_species_manuscript.tex [hunk 1]
```diff
@@ -377,9 +387,9 @@ \subsubsection{Implementing a population genomic
 we hope that many more will be available in the near future. This is on
 the horizon\\
```
Comment (lauterbur) [inline adding_species_manuscript.tex]: does this line break need to be here?

File: adding_species_manuscript.tex [hunk 2]
```diff
@@ -0,0 +1,847 @@
...
+non-model species. Table 1 displays some considerations and solutions
+For example, if all the parameters above are available, except for a
+chromosome level assembly, then one might choose to include only the
+larger contigs.
+
+Table 1. Guide to missing parameters.
+
+% Please add the following required packages to your document preamble:
+% \usepackage{booktabs}
+\begin{table}[]
```
Comment (lauterbur) [inline adding_species_manuscript.tex:530]: table needs linebreaks, but should these be added later when the content is finalized?

**Review by lauterbur — APPROVED** [review]

**Comment by lauterbur** [comment]
Awesome, thanks for doing that! I'm happy with deleting the deprecated .md file.

File: adding_species_manuscript.tex (hunk 1 above)
Comment (andrewkern) [reply adding_species_manuscript.tex]: nope

File: adding_species_manuscript.tex (hunk 2 above)
Comment (andrewkern) [reply adding_species_manuscript.tex:530]: we can deal with these later when we have the content there

**Comment by andrewkern** [comment]
okay @lauterbur I made these edits. I'm going to squash and merge my edits in and then folks can edit the tex file directly

**Pull request merged**